build.gradle

```diff
@@ -131,6 +133,13 @@ task cmake(type: Exec) {
         if (hasProperty('toolchain')) {
             toolchain = project.toolchain
         }
+        if (project.platform == "linux-arm"  && project.hasProperty("armSuffix")) {
+            toolchain = "platforms/linux/arm-gnueabihf.toolchain.cmake"
+            println "Using toolchain '${toolchain}'"
+            return defaultCmakeArgs + "-DCMAKE_TOOLCHAIN_FILE=${toolchain}" +
+                '-DENABLE_NEON=ON' +
```

Comment:
It messes up the outputs, because they are not designed for both shared and static right now. Also wanted to match the dt builds, where we only build static. Basically all our repos build and link statically for everything but roborio builds, including cscore and ntcore on non roborio arm builds.